ingestion/pom.xml

```diff
@@ -155,12 +108,24 @@
       <version>3.0.0</version>
     </dependency>
 
+    <dependency>
+      <groupId>javax.validation</groupId>
+      <artifactId>validation-api</artifactId>
+      <version>2.0.1.Final</version>
+    </dependency>
+
     <dependency>
       <groupId>org.hibernate.validator</groupId>
       <artifactId>hibernate-validator</artifactId>
       <version>6.0.13.Final</version>
     </dependency>
 
+    <!-- To force version Google Cloud SDKs need, over older that grpc does -->
+    <dependency>
+      <groupId>io.opencensus</groupId>
+      <artifactId>opencensus-contrib-http-util</artifactId>
+    </dependency>
```

Comment:
Didn't catch it until now but dependency analyzer says this is unused in Ingestion since @zhilingc removed the 0.1 legacy `PathUtil` earlier today. Also direct use of `kafka-clients` is gone since today, only Beam APIs for Kafka. Updating the last commit with removals of these, sorry for one more force-push.